### PR TITLE
Milestone2: Query and Update Animals table

### PR DESCRIPTION
**Pull request by carlylechia**
In order to complete this milestone, I:

- [x] Added a column, 'species' of type, 'string' to my `animals table` and modified my `schema.sql file`
- [x] Inserted 7 additional rows of data and modified my `data.sql file`
- [x] Inside a transaction, updated the `animals table` by setting the `species column` to *unspecified*. Verified that the change was made. Then rolled back the change and verified that the `species columns` went back to the state before the transaction.
- [x] Created two more transactions and updated the `animals table` according to the demands in the [project requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md)
- [x] All the updates made were recorded into the `data.sql file`
- [x] Wrote queries to answer all the questions posed in the **project requirements**, each query recorded into the `queries.sql file`

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -4,4 +4,26 @@ INSERT INTO animals VALUES (1, 'Agumon', '2020-02-03', 0, true, 10.23);
 INSERT INTO animals VALUES (2, 'Gabumon', '2018-11-15', 2, true, 8);
```
Comment (mateo951) [inline data.sql:4]: Well done 💪 Data structure was followed in data, queries, and schema files 🔥

File: data.sql [hunk 2]
```diff
@@ -4,4 +4,26 @@ INSERT INTO animals VALUES (1, 'Agumon', '2020-02-03', 0, true, 10.23);
...
 INSERT INTO animals VALUES (3, 'Pikachu', '2021-01-07', 1, false, 15.04);
 INSERT INTO animals VALUES (4, 'Devimon', '2017-05-12', 5, true, 11);
+INSERT INTO animals VALUES(5, 'Charmander', '2020-02-08', 0, false, -11);
+INSERT INTO animals VALUES(6, 'Plantmon', '2021-11-15', 2, true, -5.7),(7, 'Squirtle', '1993-04-02', 3, false, -12.13), (8, 'Angemon', '2005-06-12', 1, true, -45), (9, 'Boarmon', '2005-06-7', 7, true, 20.4), (10, 'Blossom', '1998-10-13', 3, true, 17), (11, 'Ditto', '2022-05-14', 4, true, 22);
```
Comment (mateo951) [inline data.sql:8]: Correct data values are inserted into the DB ✔

File: queries.sql [hunk 3]
```diff
@@ -7,4 +7,23 @@ SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
...
+--  t        | 3.0000000000000000
+SELECT species, MAX(weight_kg), MIN(weight_kg) FROM animals GROUP BY species;
+-- species | max | min 
+-- ---------+-----+-----
+--  pokemon |  22 |  11
+--  digimon |  45 | 5.7
+SELECT species, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY species;
+-- species |        avg         
+-- ---------+--------------------
+--  pokemon | 3.0000000000000000
```
Comment (mateo951) [inline queries.sql:29]: All of your transactions are on point 🤓 Fantastic job 🔥

**Review by mateo951 — APPROVED** [review]
Hi @carlylechia, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🤯 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 3 above)
Comment (carlylechia) [reply queries.sql:29]: Thank you man

**Pull request merged**